### ISSUE TITLE
1.1.0 (ADAS) - FEATURE - enable security - Kerberos support in SSM

### ISSUE THREAD

**Issue by aurorahunter** [Intel-bigdata/SSM#1205]
the username/password could be in a config file or in mysql

**Comment by ChenSammi** [comment]
Hi Varsha, 

Are you suggesting to enable security on SSM UI?

**Comment by aurorahunter** [comment]
Yes, to enable Kerberos on SSM

**Comment by ChenSammi** [comment]
OK. Can you rephrase the title, to emphases that enable Kerberos support?  Also what's "the WEB-Login screen popped up" refer to?

**Comment by aurorahunter** [comment]
Few weeks back when Kai was in SC, he mentioned about having a single user login using a web screen pop up when we open up SSM Gui. I dont' believe that is part of 1.1.0 branch and hence I mentioned the support for this. Again this was just a interim solution till Kerberos is officially supported
